Vendor/SweetUIKit/UIAlertController+Sweetness.swift

```diff
@@ -41,9 +41,16 @@ public extension UIAlertController {
      - parameter error: The error.
      - returns: The alert controller.
      */
-    public static func errorAlert(_ error: NSError) -> UIAlertController {
+    public static func errorAlert(_ error: NSError, closeBlock: ((Void) -> Void)? = nil) -> UIAlertController {
```

Comment:
Please, rollback this one since it's not on `UIAlertController+Sweetness.swift`.